hoomd/operation.py

```diff
@@ -394,15 +211,15 @@ def _setattr_param(self, attr, value):
                                      " initialization".format(attr))
 
     def __eq__(self, other):
-        other_keys = set(other.__dict__.keys())
+        other_keys = other.__dict__.keys()
         for key in self.__dict__.keys():
             if key in self._skip_for_equality:
```

Comment:
Wondering if `other._skip_for_equality` matters?

Reply:
It shouldn't since this is set at the class level, and objects of different classes shouldn't be equal in the first place.